helm/blog-app/templates/secrets.yaml

```
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "blog-app.fullname" . }}-secrets
+  namespace: {{ .Values.namespace.name }}
+  labels:
+    {{- include "blog-app.labels" . | nindent 4 }}
+type: Opaque
+data:
+  {{- if .Values.secrets.mongoUri }}
+  mongoUri: {{ .Values.secrets.mongoUri | b64enc }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)